src/shared/components/case-editor/case-edit-submit/case-edit-submit.html

```diff
@@ -2,9 +2,9 @@
 <h1 class="govuk-heading-l">{{ eventTrigger.name}}</h1>
 
 <!-- Case ID -->
-<h2 *ngIf="getCaseId()" class="heading-h2">
+<span *ngIf="getCaseId()" class="heading-span" title="Case reference {{ getCaseId() | ccdCaseReference}}">
   #{{ getCaseId() | ccdCaseReference}}
-</h2>
+</span>
```

Comment:
I really don't like this <span> thing... I know it is what you have been told to do. But it is such an abuse of spans

Reply:
I do think this will get flagged in future accessibility audits for poor HTML structure

Reply:
It highlights the need to move the case reference somewhere else, however for the moment 'demoting' the h2 to a span is a step in the right direction.